publishing.md

```diff
@@ -53,12 +53,12 @@ Please make sure to update the destination of the source code link if you use a
 </span>
 ```
 
-## Distributing changes
+## Changes
 
-If you modify or extend OpenFisca, and if you distribute those changes, you are [legally required](https://choosealicense.com/licenses/agpl-3.0/) to make those changes available to the community. The easiest way to do it is to publish your fork, country package, extension or reform on a source hosting platform such as GitHub, and to [notify](./community.md) the core team of this publication.
+If you modify or extend OpenFisca, you are [legally required](https://choosealicense.com/licenses/agpl-3.0/) to make those changes available to the community. The easiest way to do it is to publish your fork, extension or reform on a source hosting platform such as GitHub, and to [notify](./community.md) the core team of this publication.
 
-"Distributing changes" means sharing with others, in any way, the changes you made to the software. Providing software that integrates your changes in any format (source code, package, etc.) is considered as distribution, which means you also have to make your changes available.
+Please note that, under AGPL provisions, serving over the network is considered as publishing changes, which means you also have to make your changes available even if you don't keep the running instance on your servers.
 
-Please also note that, under AGPL provisions, serving over the network is considered as distribution, which means you also have to make your changes available.
+You can [read this](https://softwareengineering.stackexchange.com/questions/107883/agpl-what-you-can-do-and-what-you-cant/314908/) to get more insights on what you can do and what you can't with an APGL licensed software.
```

Comment:
Minor phrasing improvements:
- `to get more insight` (singular)
- `on what you can and can't do`
- `with AGPL licensed software` (remove `an`)